guides/source/active_storage_overview.md

```diff
@@ -55,14 +55,17 @@ WARNING: Before you install and use third-party software, make sure you understa
 
 Active Storage uses three tables in your application’s database named
 `active_storage_blobs`, `active_storage_variant_records`
-and `active_storage_attachments`. After creating a new
-application (or upgrading your application to Rails 5.2), run
-`bin/rails active_storage:install` to generate a migration that creates these
-tables. Use `bin/rails db:migrate` to run the migration.
+and `active_storage_attachments`. Run `bin/rails active_storage:install`
```

Comment:
Tiniest of nits -- could we order the tables in the same order they appear below, ie.
```
Active Storage uses three tables in your application’s database named
`active_storage_blobs`, `active_storage_attachments `
and `active_storage_variant_records`.
```
?

Reply:
Good call. Done.